### PR TITLE
Added NSNumberFormatter for easier number formatting

### LINKED ISSUE #1: Add `NSNumberFormatter` property for value formatting

### PR DESCRIPTION
**Pull request by BalestraPatrick**
Fixed #1 

It's now possible to customize or provide a `NSNumberFormatter` object in this way:

``` swift
let moneyFormatter = NSNumberFormatter()
moneyFormatter.numberStyle = .CurrencyStyle
moneyFormatter.maximumFractionDigits = 0
stepper.numberFormatter = moneyFormatter
```

Or simply modify the default `NSNumberFormatter`:

``` swift
stepper3.numberFormatter.maximumFractionDigits = 1
```

### CODE REVIEW

**Comment by BalestraPatrick** [comment]
@a2 Thoughts on this implementation? :grinning:

**Comment by a2** [comment]
Looks good to me. :+1:

File: Pod/Classes/ValueStepper.swift [hunk 1]
```diff
@@ -124,6 +118,7 @@ private enum Button: Int {
     
     public required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
+        translatesAutoresizingMaskIntoConstraints = false
```
Comment (a2) [inline Pod/Classes/ValueStepper.swift]: Not sure about this line. It's usually supposed to be set by the user of the class, no?

File: Pod/Classes/ValueStepper.swift [hunk 2]
```diff
@@ -279,11 +290,11 @@ private enum Button: Int {
     // MARK: Actions
     
     private func setState() {
-        if value.round1 >= maximumValue {
+        if value.rounded(numberFormatter.maximumFractionDigits) >= maximumValue {
```
Comment (a2) [inline Pod/Classes/ValueStepper.swift]: Why do you need to round the number?

File: Pod/Classes/ValueStepper.swift (hunk 1 above)
Comment (BalestraPatrick) [inline Pod/Classes/ValueStepper.swift]: You're right. I thought it was needed to get IB to pick up the `intrinsicContentSize` but it's not. I'm removing it.

Comment (a2) [inline Pod/Classes/ValueStepper.swift]: IB won't pick up the `intrinsicContentSize`. You need to set it explicitly in the storyboard.

File: Pod/Classes/ValueStepper.swift (hunk 2 above)
Comment (BalestraPatrick) [inline Pod/Classes/ValueStepper.swift]: In fact, it's not needed! :laughing:

File: Pod/Classes/ValueStepper.swift (hunk 1 above)
Comment (BalestraPatrick) [inline Pod/Classes/ValueStepper.swift]: Setting a value as the `intrinsicContentSize` in the subclass allows IB to automatically infer the width and height constraints without actually specifying them.
screen shot 2016-02-20 at 5 06 45 pm

**Pull request merged**